packages/pwa-storefront-mcp/jest.config.js

```diff
@@ -6,12 +6,15 @@
  */
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const parentConfig = require('@salesforce/pwa-kit-dev/configs/jest/jest.config.js')
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
 
 module.exports = {
     ...parentConfig,
     testEnvironment: 'node',
     testMatch: ['**/__tests__/**/*.js', '**/?(*.)+(spec|test).js'],
     testPathIgnorePatterns: ['bin/*', 'coverage/*', 'dist/*', 'node_modules/*', 'scripts/*'],
+    setupFilesAfterEnv: [path.join(__dirname, 'jest-setup.js')],
```

Comment:
Small fix: We had a jest-setup file but it wasn't wired up, so I'm doing that here.